src/assets/styles/base/_base.scss

```diff
@@ -0,0 +1,5 @@
+@import "fonts";
+
+h1 {
+    font-family: $family-sans-serif;
```

Comment:
I'd rather not use Sass here and use CSS custom properties. Start with a single file and use imports only when necessary, like here: https://github.com/inclusive-design/acaw-cama/blob/main/src/assets/styles/app.css

Reply:
Oh, this is just to have working example, I agree that we should only use imports when necessary.

Reply:
Still I don't think we should use Sass. Maybe let's discuss tomorrow.